src/bm.h

```diff
@@ -694,11 +694,13 @@ Word number_literal_as_word(String_View sv)
 void bm_translate_source(String_View source, Bm *bm, Basm *basm)
 {
     bm->program_size = 0;
+    int line_number = 0;
 
     // First pass
     while (source.count > 0) {
         assert(bm->program_size < BM_PROGRAM_CAPACITY);
         String_View line = sv_trim(sv_chop_by_delim(&source, '\n'));
+        line_number += 1;
```

Comment:
Small suggestion, could you add the file name into the PR ?
On line 825 / 826 you could display "ERROR: fib.basm:15 - Unknown instruction `must`".

So it could be used with ["compilation mode" tool](https://github.com/tsoding/bm) to jump right back into the editor.

Reply:
I tried, but was not sure. I've read the issue second time and implemented as you suggested!